script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -865,10 +875,10 @@ In an instant, the atmosphere reverts back to its original temperature.
 -- 自然現象とはかけ離れた寒波は、ヤツの命と共に消失する。
 The cold wave, far removed from any natural phenomenon, disappears along with the man's life.
 }
-[sha:dad68dc0d57ba4fe53a4ebc71004f2cdb270f5ab]{
+[offset:10581]{
 -- Page 710, Offset 10581.
 -- その中で、
-In the middle of it all―――
+And in its midst―――
```

Comment:
I'd probably render this as temporal, since the line above says the mist is gone
> And as it does―――